packages/story-editor/src/utils/useCORSProxy.ts

```diff
@@ -64,9 +64,9 @@ function useCORSProxy() {
    * @return {null|string} Return proxied source or null.
    */
   const getProxiedUrl = useCallback(
-    (resource, src) => {
+    (resource: Resource, src: string) => {
       const { needsProxy } = resource;
-      if (needsProxy && src) {
+      if (needsProxy && src && typeof getProxyUrl === 'function') {
```

Comment:
```suggestion
      if (needsProxy && src && getProxyUrl) {
```

This should be enough.

See similar change in #12763